kernelci/build.py

```diff
@@ -802,6 +805,22 @@ def _kernel_config_enabled(self, config_name):
         cmd = 'grep -cq CONFIG_{}=y {}'.format(config_name, dot_config)
         return shell_cmd(cmd, True)
 
+    def _kernel_config_getkey(self, opt_name):
+        dot_config = os.path.join(self._output_path, '.config')
+        with open(dot_config, 'r') as fp:
+            for line in fp:
+                if line.startswith(opt_name):
```

Comment:
I don't think this check is needed. In line 814 you check if this is a right option.

Reply:
Thats suggested by @gctucker : https://github.com/kernelci/kernelci-core/pull/1407#discussion_r963348987

Reply:
That's to avoid splitting all the lines.  But a regex would be ideal.

Reply:
I'm not entirely sure how much of a performance improvement we get  here, while we're  definitely  having nested conditions, but I guess we can leave it as is.